src/status_im/ui/components/chat_icon/screen.cljs

```diff
@@ -16,11 +16,12 @@
     (first name)]])
 
 (defn chat-icon [photo-path {:keys [size accessibility-label]}]
-  (let [photo (if (s/starts-with? photo-path "contacts://")
-                (->> (s/replace photo-path #"contacts://" "")
-                     (keyword)
-                     (get resources/contacts))
-                {:uri photo-path})]
+  (if-let [photo (when photo-path
+                   (if (s/starts-with? photo-path "contacts://")
+                     (->> (s/replace photo-path #"contacts://" "")
+                          (keyword)
+                          (get resources/contacts))
+                     {:uri photo-path}))]
     [image {:source photo
```

Comment:
What happens if photo is nil?

Reply:
This actually means that screen wasn't opened yet, so nothing. The thing is that component is rendered before it is shown and that's why `photo-path` can be `nil`. Without this check `s/starts-with?` may throw exception.